arch/arm/boot/dts/overlays/README

```diff
@@ -1045,10 +1045,32 @@ Load:   dtoverlay=hifiberry-amp
 Params: <None>
 
 
-Name:   hifiberry-dac
-Info:   Configures the HifiBerry DAC audio card
-Load:   dtoverlay=hifiberry-dac
-Params: <None>
+Name:   hifiberry-amp100
+Info:   Configures the HifiBerry AMP100 audio card
+Load:   dtoverlay=hifiberry-dacplus,<param>=<val>
```

Comment:
The example needs to match the overlay name.

Reply:
ok, will be fixed